dmoj/executors/NETCS.py

```diff
@@ -0,0 +1,45 @@
+import os
+
+from dmoj.cptbox.filesystem_policies import ExactFile, RecursiveDir
+from dmoj.executors.compiled_executor import CompiledExecutor
+
+CSPROJ = b"""\
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>net6.0</TargetFramework>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <Nullable>enable</Nullable>
+    <PublishSingleFile>true</PublishSingleFile>
+  </PropertyGroup>
+</Project>
+"""
+
+HELLO_WORLD_PROGRAM = """\
+Console.WriteLine("Hello, World!");
```

Comment:
This program should echo input to output, not just write out "Hello, World!". See e.g. https://github.com/DMOJ/judge-server/blob/master/dmoj/executors/C.py#L11

Reply:
Is Rust's program wrong then? https://github.com/DMOJ/judge-server/blob/master/dmoj/executors/RUST.py#L78

Reply:
I'll update it sometime later then.

Reply:
👍 will change the behavior